tools/standalone-binary-distro/make_distro_builds.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/zsh
+
+gh auth status;
+if [[ $? != 0 ]]; then
+  exit 1
+fi
+
+SDK_VERSION=$(git describe --exact-match --tags)
+if [[ $? != 0 ]]; then
+  echo "❌ Aborting: HEAD doesn't have a tag!"  
+  exit 2
```

Comment:
i didn't understand what you tried to solve here 🤔 can you explain a bit more?

this script can be run outside of `make ship` command too and you must be in a commit that is tagged while running this script, otherwise you may create binaries from `commit1` and upload it to `release2`

Reply:
Ah I see. So this indeed requires fetching the tag from remote - makes sense for the consistency 👍.